blocks/library/latest-posts/index.js

```diff
@@ -99,13 +99,17 @@ registerBlockType( 'core/latest-posts', {
 			const { latestPosts } = this.state;
 			const { setAttributes } = this.props;
 
-			if ( ! latestPosts.length ) {
+			const hasPosts = Array.isArray( latestPosts ) && latestPosts.length;
+			if ( ! hasPosts ) {
```

Comment:
Hi @Soean, if there are no posts we just render the placeholder. So if the user has no posts the user will not be able to change to block settings e.g: number of posts to display. I think if the user does not have posts he should be able to use the inspector to configure the block like if posts existed.

Reply:
@jorgefilipecosta You are right. Now the inspector controls are also shown on the placeholder view. The BlockControls with align and layoutControls are only visilble if posts existed.

Reply:
Hi @Soean this tests well for me! Nice work :) @mcsf, @gziolo could you have a second look 
 👀  before merging?